### PR TITLE
CORS filter

### PR DESCRIPTION
**Pull request by Subasinghe**

### CODE REVIEW

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/security/CrossDomainFilter.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
+ *  Copyright (c) 2015 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vinok88) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/security/CrossDomainFilter.java]: Fix the licence year.

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/security/CrossDomainFilter.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+package org.wso2.carbon.bpmn.rest.security;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+import java.io.IOException;
+
+/**
```
Comment (vinok88) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/security/CrossDomainFilter.java]: Add doc comments here.

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/ProcessAndTaskService.java [hunk 3]
```diff
@@ -0,0 +1,490 @@
+/**
+ *  Copyright (c) 2015 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vinok88) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/ProcessAndTaskService.java]: Fix the year.

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/security/CrossDomainFilter.java (hunk 2 above)
Comment (Subasinghe) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/security/CrossDomainFilter.java]: already fixed.

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java [hunk 4]
```diff
@@ -0,0 +1,242 @@
+/**
+ *  Copyright (c) 2015 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Drifftr) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java]: Year should be 2016

**Pull request closed without merging**